benchmarks/2019.json

```diff
@@ -4622,14 +4850,14 @@
     "benchmarkYear": 2019,
     "submissionMethod": "registry",
     "deciles": [
-      96.4286,
+      91.0237,
       62.2222,
-      53.8961,
```

Comment:
In this case, the performance benchmark is overriding the historical benchmark which is *not* okay. In the case a historical benchmark exists, the performance benchmark is not applied. @branonbarrett

Reply:
`"benchmarkYear": 2019,` indicates this is not a historical benchmark right?